security-analysis/security-analysis-api/src/main/java/com/powsybl/security/detectors/AbstractContingencyBlindDetector.java

```diff
@@ -50,9 +63,25 @@ public void checkCurrent(Branch branch, Consumer<LimitViolation> consumer) {
         checkCurrent(branch, Branch.Side.TWO, consumer);
     }
 
+    @Override
+    public void checkCurrentDc(Branch branch, double dcPowerFactor, Consumer<LimitViolation> consumer) {
+        checkCurrentDc(branch, Branch.Side.ONE, dcPowerFactor, consumer);
+        checkCurrentDc(branch, Branch.Side.TWO, dcPowerFactor, consumer);
+    }
+
     @Override
     public void checkAll(Network network, Consumer<LimitViolation> consumer) {
         network.getBranchStream().forEach(b -> checkCurrent(b, consumer));
+        checkVoltageLevels(network, consumer);
+    }
+
+    @Override
+    public void checkAllDc(Network network, double dcPowerFactor, Consumer<LimitViolation> consumer) {
+        network.getBranchStream().forEach(b -> checkCurrentDc(b, dcPowerFactor, consumer));
+        checkVoltageLevels(network, consumer);
```

Comment:
To remove because there is no voltage magnitude calculation in DC LF calculation